### ISSUE TITLE
Pass a parameter to COUNT?

### ISSUE THREAD

**Issue by wilson428** [kevinschaul/binify#21]
In many cases, I have a series of points that are already aggregate values of some data point. For example, I might have a point for every county with an attached field in the .shp file for jobs created in that county in the past month. 

My goal in this case would be to create hexagonal bins where COUNT = jobs in that hexagon. Would it be possible for COUNT to represent not just an actual count but a sum of a given field?

**Comment by kevinschaul** [comment]
Makes sense. Any solution might as well fix issue #8, too. I'm trying to envision how such an option would be structured.

My initial thought isn't very good:
`--accumulate-type sum --accumulate-column FIELD`

Suggestions?

**Comment by wilson428** [comment]
I think that would work just fine. There should probably still be a raw COUNT field that always means the same thing, and this flag would add a data property with the name of FIELD and the value of the sum (or whatever else it is).
